arkouda/pdarraycreation.py

```diff
@@ -459,21 +462,21 @@ def arange(*args) -> pdarray:
     >>> ak.arange(-5, -10, -1)
     array([-5, -6, -7, -8, -9])
     """
-   
-    #if one arg is given then arg is stop
+    from arkouda.numeric import cast
+    # if one arg is given then arg is stop
     if len(args) == 1:
         start = 0
         stop = args[0]
         stride = 1
 
-    #if two args are given then first arg is start and second is stop
+    # if two args are given then first arg is start and second is stop
     if len(args) == 2:
         start = args[0]
         stop = args[1]
         stride = 1
 
-    #if three args are given then first arg is start,
-    #second is stop, third is stride
+    # if three args are given then first arg is start,
+    # second is stop, third is stride
     if len(args) == 3:
         start = args[0]
         stop = args[1]
```

Comment:
Stride is never set here. stride = args[2] should be added.

Reply:
I was really confused for a second lol. Stride is set, it's just collapsed on github